Reference/Searching/Examine/quick-start.md

```diff
@@ -1,30 +1,31 @@
 # Quick start
 
-**Applies to: Umbraco 4.11.5+**
-
-_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 2 internal indexes which should not be used for searching when returning results on a public website because it indexes content that has not been published yet. So to get started we need to create a new index._
+**Applies to: Umbraco 7+**
```

Comment:
This is incorrect.  Examine is introduced somewhere in v4.
If there is anything particular about v4 which is not applicable to v6 or v7, we could move to another file (quick-start.v-4.11.5.md).  Like that it will be picked up later on by a new OUR feature.

Love to chat about that.

Reply:
Makes sense - I don't believe there's anything specific - the reason I changed it is so that users don't automatically think it's out of date, or assume there's something that's more recent elsewhere. We could probably remove this line altogether as the samples haven't changed and so ultimately are applicable across versions?

Reply:
I understand why you did.  If it's still current we can just remove the version indeed.
We will be following this path more in the future.  The current version is current.  If there is information about old versions we will be moving this information to other files.